_sass/main.scss

```diff
@@ -49,8 +51,38 @@ figcaption {
     text-align:center;
 }
 
+figure {
+    padding-bottom: 0.5rem;
+}
+
 // FOOTER
 
+@media screen and (max-width: $small) {
+
+    .footer-col {
```

Comment:
can we use this pattern for responsive styling? It's similar to what is in the header CSS

.footer-col {
  // default styles (on mobile)

    @media screen and (min-width: $on-medium) {
        // large screen styles
    }

}



}